### DISCUSSION TITLE
Discussion templates

### DISCUSSION THREAD

**Discussion by marcastel** [community/community#3709] [Discussions]
**Feature request**
Availability of discussion templates similar to issue templates, optionally bound to categories.

**Use cases**
- Ask me anything (AMA) template
- Request for comments (RFC) template
- Working instructions (WIN) template
- User acceptance testing (UAT) template
- Contributor onboarding questionnaire (COQ) checklist

**Comment by TechStudent10** [comment]
This seems like a good idea!

**Reply by MJP8** [reply]
I think so too!

**Reply by aahnik** [reply]
this is a very much needed feature.

**Comment by orta** [comment]
My problem: I want to let the user know about how they need to include a particular string in their message so that we can build automation on top of it. 

One possible solution to this is to add discussion templates in the same way we can for issues/pull requests 👍🏻

**Comment by markseuffert** [comment]
See also #2838

**Comment by Wladefant** [comment]
It is implemented now. See #2838 for further information. 
Here are some links: 
Read the blog post by GitHub: https://github.blog/2023-01-09-github-discussions-just-got-better-with-category-forms/

Read the docs: https://docs.github.com/en/discussions/managing-discussions-for-your-community/creating-discussion-category-forms

There's an example in the docs here: https://docs.github.com/en/discussions/managing-discussions-for-your-community/syntax-for-discussion-category-forms
